client/src/components/Layout/AppNavbar.vue

```diff
@@ -38,22 +104,59 @@ nav {
   background-color: v.$main-bkgrnd !important;
   height: 70px !important;
 
+  .v-btn {
+    padding: 0 !important;
+    text-transform: none;
+    letter-spacing: 0;
+    vertical-align: initial;
+
+    .mdi-chevron-down::before {
+      color: v.$main-white !important;
+    }
+  }
+
   .v-toolbar__content {
     height: 70px !important;
     justify-content: space-between;
   }
   &__logo {
     height: 40px;
   }
+
+  &__links {
+    display: inline-flex;
+    flex-wrap: nowrap;
+    align-items: center;
+    justify-content: space-between;
+    width: 270px;
+  }
+
   &__link {
     color: v.$main-white !important;
     font-family: v.$navbar-font;
     font-size: 1.3rem;
-    margin: auto 10px;
     text-decoration: none;
 
-    &.router-link-exact-active {
-      color: v.$accent-green !important;
+    &--menu-item {
+      width: 100%;
+      text-align: left;
+      font-size: 1.1em;
+      padding: 10px 20px;
+    }
+
+    &--menu-header {
+      margin-right: 2px;
+    }
+  }
+
+  &__menu {
+    top: 71px !important;
+
+    .v-list-item {
+      padding: 0;
+      :hover {
+        background-color: v.$main-grey-underlay !important;
+      }
     }
   }
```

Comment:
I think that we should also add a top border to this div so that it aligns more with Techtonic's navbar